internal/venice-common/src/test/java/com/linkedin/venice/utils/UtilsTest.java

```diff
@@ -167,20 +167,36 @@ public void testIterateOnMapOfLists() throws Exception {
 
   @Test
   public void testParseMap() {
-    assertEquals(Utils.parseCommaSeparatedStringMapFromString("", "test_field").size(), 0);
-    Map nonEmptyMap = Utils.parseCommaSeparatedStringMapFromString("a=b", "test_field");
+    assertEquals(Utils.parseJsonMapFromString("", "test_field").size(), 0);
+    Map nonEmptyMap = Utils.parseJsonMapFromString("a=b", "test_field");
     Map expectedMap = new HashMap<>();
     expectedMap.put("a", "b");
     assertEquals(nonEmptyMap, expectedMap);
 
-    VeniceException e = expectThrows(
-        VeniceException.class,
-        () -> Utils.parseCommaSeparatedStringMapFromString("invalid_value", "test_field"));
+    VeniceException e =
+        expectThrows(VeniceException.class, () -> Utils.parseJsonMapFromString("invalid_value", "test_field"));
     assertTrue(e.getMessage().contains("must be key value pairs separated by comma"));
   }
 
   @Test
   public void testSanitizingStringForLogger() {
     Assert.assertEquals(Utils.getSanitizedStringForLogger(".abc.123."), "_abc_123_");
   }
+
+  @Test
+  public void testParseJsonMapFromString() {
+    try {
+      Utils.parseJsonMapFromString(
+          "{\"changeCaptureView\": {\"viewClassName\": \"com.linkedin.venice.views.ChangeCaptureView\",\"params\": {}}}",
+          "someField"); // Method that should not throw an exception
+    } catch (Exception e) {
+      fail("Expected no exception to be thrown, but got: " + e.getMessage());
+    }
+
+    Assert.assertThrows(
```

Comment:
Can you add a comment about why this throws an exception? I guess it is because the string is not a valid JSON, right?